custom_plugin/custom_plugin.py

```diff
@@ -42,7 +42,7 @@ def gen_content_from_macros():
             mod_name = f"{scripts}.{stem}"
             mod = importlib.import_module(mod_name)
             fun = getattr(mod, stem)
-            outputs[fn] = fun()
+            outputs[fn[:-3]] = fun()
```

Comment:
Hmm, can you explain why this fixes the bug exactly?

Why `3` and not `2` or `5`?

Seems like we're hardcoding something here that may change and will result in breaking the macros again...

Reply:
I that case I would make this more explicit, by i) adding a comment to explain why it's needed, ii) using `os.path.splitext` instead:

```python
>>> os.path.splitext('foo.py')
('foo', '.py')
>>> os.path.splitext('foo.py')[0]
'foo'